centreon/src/Security/TokenAPIAuthenticator.php

```diff
@@ -166,7 +157,10 @@ private function getUserAndUpdateToken(string $apiToken): UserInterface
             throw new ContactDisabledException();
         }
 
-        $this->authenticationRepository->updateProviderTokenExpirationDate($providerToken);
+        $isTokenManual = $this->readTokenRepository->isTokenTypeManual($apiToken);
+        if (! $isTokenManual) {
+            $this->authenticationRepository->updateProviderTokenExpirationDate($providerToken);
+        }
```

Comment:
```suggestion
        if (! $this->readTokenRepository->isTokenTypeManual($apiToken)) {
            $this->authenticationRepository->updateProviderTokenExpirationDate($providerToken);
        }
```

if isTokenManual is only used once, there is no needs to use a variable, the method name is clear enough to undertand the behaviour

Reply:
hummm for the debugging and the code reading it's more clearer to have the result of the method in the variable ;)